cdi-platform-tck/cdi-platform-tck-docker-runner/pom.xml

```diff
@@ -0,0 +1,349 @@
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.jakarta.tests.tck</groupId>
+        <artifactId>cdi-platform-tck-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cdi-platform-tck-docker-runner</artifactId>
+    <name>TCK: CDI Platform (Docker)</name>
+
+    <packaging>pom</packaging>
+
+    <description>This verifies the compliance of Payara using the Jakarta CDI Platform standalone TCK within a Docker container, to avoid issues caused with the selenium tests and more recent versions of Chrome</description>
+
+    <properties>
+        <!-- This is the maven command which the run-tck.sh script will execute within the container - -->
+        <maven.command>mvn help:active-profiles clean verify -Ppayara-ci-managed -Dtck.mode=platform -Dpayara.artifact=${payara.artifact}</maven.command>
+        <skipDockerBuild>${skipTests}</skipDockerBuild>
+        <docker.image.pull.policy>IfNotPresent</docker.image.pull.policy>
+        <skipDockerCleanup>${skipTests}</skipDockerCleanup>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>fish.payara.jakarta.tests.tck</groupId>
+            <artifactId>jakarta-platform-tck</artifactId>
+            <type>zip</type>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>fish.payara.distributions</groupId>
+            <artifactId>${payara.artifact}</artifactId>
+            <type>zip</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <skip>${skipDockerBuild}</skip>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>fish.payara.jakarta.tests.tck</groupId>
+                                    <artifactId>jakarta-platform-tck</artifactId>
+                                    <type>zip</type>
+                                    <version>${project.version}</version>
+                                    <overWrite>false</overWrite>
+                                    <outputDirectory>${project.build.directory}</outputDirectory>
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
```

Comment:
We don't install the zip
We install the individual artefacts

You need to replace this with copying the individual items that we need.

Reply:
> We don't install the zip We install the individual artefacts
> 
> You need to replace this with copying the individual items that we need.

I changed this part, to copy: 
```
<copy todir="${project.build.directory}/tck-runner/src/test/java/org/jboss/weld/tck/glassfish" overwrite="true">
                                    <fileset dir="../cdi-platform-tck-runner/src/test/java/org/jboss/weld/tck/glassfish" includes="**/*.java"/>
                                </copy>

                                <copy todir="${project.build.directory}/tck-runner/src/test/resources" overwrite="true">
                                    <fileset dir="../cdi-platform-tck-runner/src/test/resources" includes="**/*"/>
                                </copy>
```

Reply:
Also added in dockerfile: RUN mvn clean verify -pl cdi-platform-tck/ -Dpayara.version=@payara.version@ -Ppayara-server-managed,jakarta-staging